frontend/src/metabase/visualizations/click-actions/Mode/Mode.ts

```diff
@@ -43,4 +47,14 @@ export class Mode {
       return actions;
     }
   }
+
+  isDrillEnabled = (drill: DrillThruDisplayInfo): boolean => {
```

Comment:
Can you explain why we need this function now? Why does it work okay without this function and we only need this when we're adding drills to SDK?

The API looks a bit out of place. Inside `queryDrill` we call 

https://github.com/metabase/metabase/blob/c91b8e4c5761d01bb75e1a50de5c5363285986d8/frontend/src/metabase/querying/utils/drills/query-drill.ts#L16

which the name suggests these are the drills available for this particular query.

As I understand, this might be a workaround until all drills for the embedding SDK are supported and has the same feature parity as our app, after that this function could be removed is that it?